verticapy/utilities.py

```diff
@@ -502,15 +502,33 @@ def pandas_to_vertica(
         tmp_name = ""
     path = "{}.csv".format(name)
     try:
-        df.to_csv(path, index=False)
+        str_cols = []
+        for c in df.columns:
+            if df[c].dtype == object and isinstance(df[c].loc[df[c].first_valid_index()], str):
+                str_cols += [c]
+        if str_cols:
+            tmp_df = df.copy()
+            for c in str_cols:
+                tmp_df[c] = '"' + tmp_df[c] + '"'
+            clear = True
+        else:
+            tmp_df = df
+            clear = False
+        import csv
+
+        tmp_df.to_csv(path, index=False, quoting=csv.QUOTE_NONE, quotechar='', escapechar='\\',)
```

Comment:
It would be very nice if you add a line of comment to explain what is the reason of the above changes.

Reply:
Did it!